components/text-editor/src/index.tsx

```diff
@@ -50,15 +54,10 @@ const TextEditor = ({
 	};
 
 	return (
-		<div className={wrapperClass || "w-full bg-gray-50 cursor-text "}>
+		<div className={wrapperClass || "w-full"}>
```

Comment:
Use **twMerge** here to override the class name when required. In this method if the user want to make background red they will have to pass class `w-full bg-red-400` as the w-full class will be overidden.
```suggestion
		<div className={twMerge("w-full",wrapperClass)}>
```